kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -3,24 +3,23 @@
   <core-base :topLevelPageName="topLevelPageName">
     <div slot="app-bar-actions">
       <channel-switcher @switch="switchChannel"/>
-
       <ui-icon-button
         icon="search"
         type="secondary"
         color="white"
         :ariaLabel="$tr('search')"
         @click="toggleSearch"/>
     </div>
-    <breadcrumbs slot="above" class="breadcrumbs"/>
-    <component class="content" slot="content" :is="currentPage"/>
+    <div slot="content">
+      <section-nav/>
+      <breadcrumbs class="breadcrumbs"/>
```

Comment:
Remove the class `breadcrumbs` from here, as the breadcrumbs component already has this class applied.